internal/bminventory/inventory.go

```diff
@@ -423,10 +423,27 @@ func (b *bareMetalInventory) validateRegisterClusterInternalParams(params *insta
 		if err != nil {
 			return common.NewApiError(http.StatusBadRequest, err)
 		}
-		log.Infof("HA mode is None, setting UserManagedNetworking to true and VipDhcpAllocation to false")
-		params.NewClusterParams.UserManagedNetworking = swag.Bool(true)
-		// in case of single node VipDhcpAllocation should be always false
-		params.NewClusterParams.VipDhcpAllocation = swag.Bool(false)
+
+		if params.NewClusterParams.UserManagedNetworking == nil {
```

Comment:
in case you add new checks please move it to the function
something like validateSingleNodeParams

Reply:
Please check that UI will not be broken

Reply:
sure, moved to a new func.